terraform/applications/nginx-waf/nginx/conf.d/default.conf

```diff
@@ -104,6 +104,16 @@ server {
     # redirect to homepage usa.gov
   }
 
+  #Required by Drupal/S3FS Module
```

Comment:
@casey-rapnicki-bixal can you double check this and let me know if this is the correct syntax and application for our site in order to have these s3 paths for image styles looked at by Drupal instead of sending to the s3 bucket.